literature.bib

```diff
@@ -235,20 +118,130 @@ @mastersthesis{Tschada21
 	note = {To appear}
 }
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2023 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+@article{RBP+:LMCS23,
+	author = {Runge, Tobias and Bordis, Tabea and Potanin, Alex and Th\"um, Thomas and Schaefer, Ina},
+	title = {{Flexible Correct-by-Construction Programming}},
+	journal = LMCS,
+	volume = 19,
+	issue = 2,
+	pages = {16:1--16:36},
+	keywords = {Traits, Correctness-by-Construction, Formal Methods, Post-Hoc Verification},
+	doi = {10.46298/lmcs-19(2:16)2023},
+	url = {https://lmcs.episciences.org/11444/pdf},
+	month = JUN,
+	year = 2023
+}
+
 @mastersthesis{Moosherr23,
 	author = {Moosherr, Benjamin},
 	title = {{Constructing Variation Diffs Using Tree Diffing Algorithms}},
 	school = UniUlm,
 	address = {Germany},
 	type = {Bachelor's Thesis},
+	doi = {10.18725/OPARU-50108},
+	url = {https://oparu.uni-ulm.de/xmlui/bitstream/handle/123456789/50184/BA_Moosherr.pdf},
 	typo3Tags = {SupervisorTT, SupervisorPB, VariantSyncMT},
 	month = APR,
-	year = 2023,
-	note = {To appear}
+	year = 2023
 }
 
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2023 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+@inproceedings{KKL+:SPLC23,
```

Comment:
Exactly 4 authors. Should be KKLS then right?

Reply:
Thanks, this went unnoticed in the last review :)